### ISSUE TITLE
Using filter statements fails for volume="/"

### ISSUE THREAD

**Issue by mmFooD** [digint/btrbk#209]
I am having an issue when it comes to listing snapshots from a specific subvolume. As can be seen below I have two volumes in my config file. 
```
# btrbk list config
source_host  source_subvol      snapshot_path                  snapshot_name  target_host  target_path
-----------  -----------------  -----------------------------  -------------  -----------  --------------------------
-            //etc              //.snapshots/etc               etc            pi           /backup/asus-laptop/etc
-            //var              //.snapshots/var               var            pi           /backup/asus-laptop/var
-            //opt              //.snapshots/opt               opt            pi           /backup/asus-laptop/opt
-            //srv              //.snapshots/srv               srv            pi           /backup/asus-laptop/srv
-            //root             //.snapshots/root              root           pi           /backup/asus-laptop/root
-            /mnt/rootfs/@      /mnt/rootfs/@snapshots/rootfs  rootfs         pi           /backup/asus-laptop/rootfs
-            /mnt/rootfs/@home  /mnt/rootfs/@snapshots/home    home           pi           /backup/asus-laptop/home
-            /mnt/rootfs/@vm    /mnt/rootfs/@snapshots/vm      vm             pi           /backup/asus-laptop/VM
```

Listing snapshots for individual subvolumes works fine for the subvolumes in the `/mnt/rootfs` volume. But the ones in the `/` volume is not working as expected. I get error messages like 
```
# btrbk list snapshots //etc
ERROR: Command line argument does not match any volume, subvolume, target or group declaration: /etc
```
no matter what argument I try (only 'etc', escaping the backslashes etc.).

So to make it easier to select them I tried adding these subvolumes to individual groups (see config file). But now the list only contains snapshots created previously, by snapper (which creates them in a `.snapshot` directory **inside** the subvolume itself).

```
# btrbk list snapshots etc
source_host  source_subvol  snapshot_subvol                status  target_host  target_subvol
-----------  -------------  -----------------------------  ------  -----------  -------------
-            //etc          //etc/.snapshots/1/snapshot    -       -            -
-            //etc          //etc/.snapshots/2/snapshot    -       -            -
```

**Note:** It is only **listing** them that isn't working correctly, creating the actual snapshots/backups works just fine: 
```
# ls /.snapshots/etc
etc.20180131T1707  etc.20180201T1022  etc.20180201T2122  etc.20180202T0200
```

Am I missing some important line in the docs, or is this a bug? Here are my BTRFS layout and config file:

**Subvolume layout (mountpoints in parenthesis)**
```
- ROOT (id = 5) (/mnt/rootfs/)
  \ 
   |- @ ( / )
       \ 
        |- etc
        |- opt
             .....
   |- @home (/home)
   |- @snapshots (/.snapshots)
        \ 
        |- etc
        |- opt
             .....
   |- @vm (/VM)
```

```
# /etc/btrbk/btrbk.conf

# Enable transaction log
transaction_log            /var/log/btrbk.log

# Enable lock file support: Ensures that only one instance of btrbk
# can be run at a time.
lockfile                   /var/lock/btrbk.lock

# Always create snapshots. Set this to "ondemand" to only create
# snapshots if the target volume is reachable. Set this to "no" if
# snapshot creation is done by another instance of btrbk.
snapshot_create            always

# Perform incremental backups (set to "strict" if you want to prevent
# creation of non-incremental backups if no parent is found).
incremental                yes

ssh_identity /etc/btrbk/ssh/id_rsa

timestamp_format long

# Preserve all snapshots for a minimum period of time.
snapshot_preserve_min      12h

# Retention policy for the source snapshots.
snapshot_preserve          24h 10d 1m

# Preserve all backup targets for a minimum period of time.
target_preserve_min        no

# Retention policy for backup targets:
target_preserve            20d 10w *m

volume /
    subvolume etc
        group etc
        snapshot_dir               .snapshots/etc
        target send-receive pi:/backup/asus-laptop/etc

    subvolume var
        group var
        snapshot_dir              .snapshots/var
        target send-receive pi:/backup/asus-laptop/var

    subvolume opt
        group opt
        snapshot_dir               .snapshots/opt
        target send-receive pi:/backup/asus-laptop/opt

    subvolume srv
        group srv
        snapshot_dir               .snapshots/srv
        target send-receive pi:/backup/asus-laptop/srv

    subvolume root
        group root
        snapshot_dir               .snapshots/root
        target send-receive pi:/backup/asus-laptop/root

volume /mnt/rootfs/
    subvolume @
        group rootfs
        snapshot_name rootfs
        snapshot_dir               @snapshots/rootfs
        target send-receive pi:/backup/asus-laptop/rootfs

    subvolume @home
        group home
        snapshot_name home
        snapshot_dir               @snapshots/home
        target send-receive pi:/backup/asus-laptop/home

    subvolume @vm
        group vm
        snapshot_name vm
        snapshot_dir               @snapshots/vm
        target send-receive pi:/backup/asus-laptop/VM
```

**Comment by digint** [comment]
The problem was not the listing, but the filter statements, which fail to match for `volume /`.

fixed in a3641cff74c8835bd148daa10c8ee5ee3d921ff0

**Comment by digint** [comment]
fixed in [btrbk-0.26.1](/digint/btrbk/releases/tag/v0.26.1)
